godtools/App/Share/Data/StringSearcher/StringSearcher.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  StringSearcher.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 11/7/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+
+class StringSearcher {
+    
+    func search<T: StringSearchable>(for searchText: String, in searchables: [T]) -> [T] {
```

Comment:
Nice I like the utilization of generics :)

Reply:
I love generics 😄